internal/cmd/kafka/command_mirror_promote.go

```diff
@@ -14,10 +14,11 @@ import (
 
 func (c *mirrorCommand) newPromoteCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "promote <destination-topic-1> [destination-topic-2] ... [destination-topic-N] --link my-link",
-		Short: "Promote mirror topics.",
-		RunE:  c.promote,
-		Args:  cobra.MinimumNArgs(1),
+		Use:               "promote <destination-topic-1> [destination-topic-2] ... [destination-topic-N] --link my-link",
```

Comment:
Can you do a global search over the whole repo so we're not missing any others?
```suggestion
		Use:               "promote <destination-topic-1> [destination-topic-2] ... [destination-topic-N]",
```

Reply:
Yup, I'll bundle that in with the followup PR to rename test functions.